WordPress/src/main/java/org/wordpress/android/ui/domains/DomainSuggestionsFragment.kt

```diff
@@ -62,6 +64,7 @@ class DomainSuggestionsFragment : Fragment() {
         domain_suggestions_list.layoutManager = LinearLayoutManager(activity, RecyclerView.VERTICAL, false)
         domain_suggestions_list.setEmptyView(actionableEmptyView)
         chose_domain_button.setOnClickListener {
+            AnalyticsTracker.track(Stat.DOMAIN_CREDIT_NAME_SELECTED)
```

Comment:
what do you think about moving this into the `selectDomain` method and using the `AnalyticsTrackerWrapper` instead? that way you can move it out of the fragment and test it (if you want to).